core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -36,6 +36,21 @@
 public class CustomExceptionHandler extends ResponseEntityExceptionHandler {
     private ErrorAttributes errorAttributes;
 
+    /**
+     * Method intercept exception {@link BadRequestException}.
+     *
+     * @param ex      Exception witch should be intercepted.
+     * @param request contain  detail about occur exception
+     * @return ResponseEntity witch  contain http status and body  with message of exception.
+     * @author Yurii Savchenko
+     */
+    @ExceptionHandler(BadRequestException.class)
+    public final ResponseEntity<Object> handleBadRequestException(BadRequestException ex, WebRequest request) {
+        ExceptionResponse exceptionResponse = new ExceptionResponse(getErrorAttributes(request));
+        log.trace(ex.getMessage(), ex);
```

Comment:
This needs a little improvment.
1. TRACE level is too low, I would use INFO.
2. The exception is handled, so no need to print the stack trace. Please remove the last parameter.
3. Please move logging to the first line: log the exception before any interaction.